docs/cards.html

```diff
@@ -0,0 +1,358 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<!-------------------------------------- CSS Styling Files ------------------------------------------------>
+<!------------------------ Components Styling ---------------------->
+    <link rel="stylesheet" href="../styles/components/cards.css">
+<!------------------------------------------------------------------>
+<!--------------------------------------------------------------------------------------------------------->
+<!--------------------------- Google Material Icons CDN link ---------------------------->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<!---------------------------------------------------------------------------------------->
+<!------------- Code Snapshots Formatting Styles ------------->    
+    <link rel="stylesheet" href="../utils/prism.css">
+<!------------------------------------------------------------>
+<!--------------------------------------------------------------------------------------------------------->
+    <title>TrishnaUI | Home</title>
+</head>
+<body>
+<div class="container">
+<!------------------------------- HEADER (Navbar) ----------------------------------------------------->
+<header class="header">
+    <nav class="navbar d-flex px-sm py-sm jc-sb ai-c" style="background-color: var(--smoke);">
+        <div class="nav-brand">
+            <h1>TrishnaUI</h1>
+        </div>
+        <div class="nav-links d-flex jc-sa">
+            <a class="px-sm py-sm">Home</a>
+            <a class="px-sm py-sm">Docs</a>
```

Comment:
If you are reusing px-sm and py-sm **both** at multiple places you can have p-sm generalized which gives sam padding on the x and y axis.